src/components/types.ts

```diff
@@ -136,6 +138,14 @@ export interface TableFormat extends DataFormat {
 
 export type TableData = TableFormat & Updates<TableFormat>
 
+export interface VegaLiteChartFormat extends DataFormat {
+  /** Follow Vega-lite's [documentation](https://vega.github.io/vega-lite/) to provide a specification object. Schema should be included in the spec. Need to use 'container' for width or height for responsive chart. */
```

Comment:
```suggestion
  /** Follow Vega-lite's [documentation](https://vega.github.io/vega-lite/) to provide a specification object. Schema should be included in the spec. Set width or height to 'container' in the spec for a responsive chart. */
```
```suggestion
  /** Follow Vega-lite's [documentation](https://vega.github.io/vega-lite/) to provide a specification object. Schema should be included in the spec. Need to use 'container' for width or height for responsive chart. */
```

Reply:
Are you suggesting having default value if width and height are not provided?

Reply:
I was suggesting that we force the chart to be responsive by adding the required properties to the spec.